lib/ansible/modules/files/unarchive.py

```diff
@@ -802,6 +822,7 @@ def main():
             list_files=dict(type='bool', default=False),
             keep_newer=dict(type='bool', default=False),
             exclude=dict(type='list', default=[]),
+            include=dict(type='list', default=[]),
```

Comment:
Sorry for taking a while to review this one! Do you mind rebasing this, updating the version added, and adding a changelog?

This option should probably be mutually exclusive with `exclude` - i.e. `AnsibleModule(..., mutually_exclusive=[('exclude', 'include',)])`